Assets/MRTK/Tools/Toolbox/MixedRealityToolboxWindow.cs

```diff
@@ -235,6 +245,31 @@ private void RenderSection(ToolboxCategory bucket)
                         }
                     }
 
+                    bool requiresCanvas = bucket.CategoryName.Contains(RequiresCanvas);
+                    if (requiresCanvas)
+                    {
+                        using (new EditorGUILayout.HorizontalScope())
+                        {
+                            dropdownIndex = EditorGUILayout.Popup(CanvasDropdownContent, dropdownIndex, dropdownValues);
+
+                            if (GUILayout.Button(RefreshButtonContent, EditorStyles.miniButton, GUILayout.Width(ReloadButtonWidth)))
+                            {
+                                FindAllMRTKCanvases();
+                            }
+                        }
+
+                        if (canvasUtilities.Length == 0)
+                        {
+                            GUIStyle CanvasWarningStyle = new GUIStyle(EditorStyles.textField)
+                            {
+                                wordWrap = true
+                            };
+                            GUILayout.TextField("These MRTK components require an MRTK Canvas Utility on one of the Unity UI Canvases in the scene.\nNone were detected. Press refresh if you recently added any.", CanvasWarningStyle);
+                        }
```

Comment:
It might not be obvious to someone new, but maybe we can add how to create an MRTK canvas vs just a Unity UI canvas.
```suggestion
                            GUILayout.TextField("These MRTK components require an MRTK Canvas Utility on one of the Unity UI Canvases in the scene.\nNone were detected. Press refresh if you recently added any \n To create a MRTK Canvas: GameObject > UI > Canvas, and under the Canvas component in the inspector, select Convert to MRTK Canvas.", CanvasWarningStyle);
                        }
```